### DISCUSSION TITLE
How to build full static with Bridge + Nitro ?

### DISCUSSION THREAD

**Discussion by gnuletik** [nuxt/nuxt#17732] [Questions]
Hi !
I'm currently trying to migrate a website to Nuxt Bridge with Nitro and stumbling on making a full static build.

When running `nuxi generate`, Nitro seems to build for SSR :
```sh
$ nuxi generate
...
ℹ Full static generation activated
ℹ Nitro preset is server
...
```

```sh
$ ls .output
nitro.json  public  server
```

No `dist/` directory is created so I tried to run `http-server .output/public` but I get 404 responses when the browser attempts to load payload.json files.

The only way to get a static build was to disable nitro
```ts
// nuxt.config.ts
...
bridge: {
  nitro: false
}
...
```

It seems that [another user have the issue](https://github.com/nuxt/framework/discussions/515#discussioncomment-1832023).

NB: I made these tests on Nuxt CLI v3.0.0-27369360.33ebb01

Was someone able to produce a static build with Nitro ?

Thanks!

**Comment by Loveemtrees** [comment]
I am having the same issue right now. This is the main feature I use nuxt for so would be grateful for a solution / any tips.

**Reply by gnuletik** [reply]
For my current projects, I finally did not upgrade after several attempts.
You may also lose IE11 support while upgrading to Nuxt Bridge, see https://github.com/nuxt-community/composition-api/discussions/585

**Reply by saulhardman** [reply]
I too had problems running `nuxi generate` with Nuxt 2 + Nuxt Bridge. I managed to work around a few of them, but the missing `payload.js` bundles was the last straw and I'm currently reverting to Nuxt 2 (v2.15.x). Hopefully others can see this thread and save themselves some time.
